### PR TITLE
Fixes from API changes - Sept 2016

### PR DESCRIPTION
**Pull request by na3d**
I have found at least one more bug, and need to run everything through manual testing to verify everything else is working properly, but want to get this out ASAP, so more than one person can work on the manager.

### CODE REVIEW

File: src/actions/api/services.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { makeFetchPage, makeFetchAll } from '~/api-store';
 
 export const UPDATE_SERVICES = '@@services/UPDATE_SERVICES';
-
+// FIXME Add in /linode/services/:id
```
Comment (SirCmpwn) [inline src/actions/api/services.js]: What's this?

File: src/linodes/components/Linode.js [hunk 2]
```diff
@@ -148,8 +149,8 @@ function renderCard(props) {
         <div className="form-group row">
           <div className="col-sm-12 content-col ip-addresses">
             <span className="label-col">IP Addresses</span>
-            <div>{linode.ips['public'].ipv4[0]}</div>
-            <div>{linode.ips['public'].ipv6}</div>
+            <div>{linode.ipv4.address}</div>
+            <div>{ipv6.substring(0, ipv6.indexOf('/'))}</div>
```
Comment (SirCmpwn) [inline src/linodes/components/Linode.js]: +1 to using the range, -1 to trimming from /

File: src/linodes/layouts/LinodeNetworking.js [hunk 3]
```diff
@@ -171,24 +168,11 @@ export class LinodeNetworking extends Component {
...
-                  </div>
+              <div className="form-group row">
+                <div className="col-sm-12 content-col right">
+                  {"No private IP addresses."}
                 </div>
-              )}
+              </div>
               <div className="form-group row">
                 <div className="col-sm-4 label-col left">
                   Link-local IP
```
Comment (SirCmpwn) [inline src/linodes/layouts/LinodeNetworking.js]: Presumably for the networking tab you'll want to add API store mappings for the IP addresses endpoints and fetch them on mount.

File: src/actions/api/services.js (hunk 1 above)
Comment (na3d) [inline src/actions/api/services.js]: Thank you!  They accidentally missed part of services when updating the docs.  I forgot to go back and remove this.

File: src/linodes/components/Linode.js (hunk 2 above)
Comment (na3d) [inline src/linodes/components/Linode.js]: Do we just want to include the /64 when listeing IPv6?

Comment (SirCmpwn) [inline src/linodes/components/Linode.js]: Sure.

File: src/linodes/layouts/LinodeNetworking.js (hunk 3 above)
Comment (na3d) [inline src/linodes/layouts/LinodeNetworking.js]: I figured the IP mappings will be a different task, since they are new endpoints.

**Pull request merged**